common/lib/dependabot/errors.rb

```diff
@@ -531,6 +538,20 @@ def initialize(source)
     end
   end
 
+  class GitAuthToken < DependabotError
```

Comment:
Can we have a more descriptive error name? E.g. `InvalidGitAuthToken` or something like that

Reply:
makes sense, ill change it